test/core/end2end/connection_refused_test.cc

```diff
@@ -123,8 +123,9 @@ static void run_test(bool wait_for_ready, bool use_service_config) {
   grpc_completion_queue_shutdown(cq);
   while (grpc_completion_queue_next(cq, gpr_inf_future(GPR_CLOCK_REALTIME),
                                     nullptr)
-             .type != GRPC_QUEUE_SHUTDOWN)
+             .type != GRPC_QUEUE_SHUTDOWN) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done